src/quemb/molbe/misc.py

```diff
@@ -393,12 +392,11 @@ def be2puffin(
                 mf = qmmm.mm_charge(
                     mf1, pts_and_charges[0], pts_and_charges[1], unit="bohr"
                 ).newton()
-                print(
+
+                raise ValueError(
                     "Setting use_df to false and jk to none: have not tested DF and "
                     "QM/MM from point charges at the same time"
                 )
```

Comment:
```suggestion
                if not use_df or not jk:
                    raise ValueError(
                        "Setting use_df to false and jk to none: have not tested DF and "
                        "QM/MM from point charges at the same time"
                    )
```

Reply:
When `use_df` is False and `jk` is None, we want QM/MM to run

Reply:
your change is different from what you are saying in words.
```python3
if not use_df and jk is None:
    # run
```
becomes
```python3
if not (not use_df and jk is None):
    # raise error
```
becomes
```python3
if use_df or jk is not None:
    # raise error
```

what do we want?

Reply:
The last code block would do! I didn't notice this before, but I suppose this input validity check & exception should happen before actually doing the mf calculations.

Reply:
thank you and done